src/_locales/en_US/messages.json

```diff
@@ -594,5 +598,13 @@
     "popup_special_page_paragraph": {
         "message": "Privacy Badger doesn't work on special pages like this one. Try browsing somewhere else.",
         "description": "Shown in the popup for special browser pages such as the New Tab page and 'about:' pages."
+    },
+    "options_widget_replacement_tab": {
+        "message": "Widget Replacement",
+        "description": "Options page tab heading"
+    },
+    "options_widget_replacement_desc": {
+        "message": "Privacy Badger can replace social buttons and other potentially useful widgets with click-to-activate placeholders.",
```

Comment:
What do you think about the following text?

>When blocking social buttons and other potentially useful (video, audio, comments) widgets, Privacy Badger can replace them with click-to-activate placeholders.

- Tries to explain "widgets"
- Tries to explain that this isn't what blocks the widgets; the widgets are blocked, and then the replacement may or may not happen.

Reply:
I like it